pkg/cloudprovider/providers/gce/gce.go

```diff
@@ -77,19 +78,25 @@ const (
 
 // GCECloud is an implementation of Interface, LoadBalancer and Instances for Google Compute Engine.
 type GCECloud struct {
+	ClusterID ClusterID
```

Comment:
Why does this need to be public?  If it does, comment.  It is different than all the others, so explain.